editor/filesystem_dock.cpp

```diff
@@ -1004,6 +1004,9 @@ void FileSystemDock::_update_file_list(bool p_keep_selection) {
 			}
 		}
 	} else {
+		if (directory.is_empty() || directory == "/") {
```

Comment:
It's still possible to break the editor with `.` and probably other strings. Maybe check whether the directory begins with `res://` and fix it if not?

Reply:
Yes thanks!